src/qonnx/custom_op/ChannelsLast/__init__.py

```diff
@@ -0,0 +1,7 @@
+from qonnx.custom_op.ChannelsLast.wrapped_ops import BatchNormalization, Conv, MaxPool
```

Comment:
to have some consistency for module names I suggest we follow the recommendations from PEP-8 https://www.python.org/dev/peps/pep-0008/#package-and-module-names 

> Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.

such that e.g. `from qonnx.custom_op.ChannelsLast.wrapped_ops import BatchNormalization, Conv, MaxPool` becomes `from qonnx.custom_op.channels_last.wrapped_ops import BatchNormalization, Conv, MaxPool`

should be checked for other module names too

Reply:
This should be addressed now.